docker-compose.yml

```diff
@@ -0,0 +1,28 @@
+version: '2'
+services:
+#Please set the environment variables before start running the container with docker compose.
```

Comment:
In the entrypoint script, if the envs are not set, fail to start with proper message

Thus people can easily debug and understand.